staging/src/k8s.io/apiserver/pkg/server/options/serving_with_loopback.go

```diff
@@ -71,7 +71,19 @@ func (s *SecureServingOptionsWithLoopback) ApplyTo(secureServingInfo **server.Se
 
 	default:
 		*loopbackClientConfig = secureLoopbackClientConfig
-		(*secureServingInfo).SNICerts[server.LoopbackClientServerNameOverride] = &tlsCert
```

Comment:
if GetConfigForClient rebuilds from SNICerts, can we leave this as-is, adding server.LoopbackClientServerNameOverride to the SNICerts map, and let GetConfigForClient rebuild from it?